client/src/Core/Components/MainLayout.tsx

```diff
@@ -91,8 +92,30 @@ const mainLayoutStyle = ({ theme }: IThemeOnlyProps): MainLayoutStyles => {
         display: 'flex',
         width: '100%',
         overflow: 'hidden',
-        height: '100%'
+        height: '100%',
+        flexDirection:'column'
       }
+    ],
+    assignmentTitle: [

```

Comment:
Correct me if I'm wrong, these lines from here onwards are not required in this file anymore right?

Reply:
Yes! Removing it in the next commit.